src/CardResult.js

```diff
@@ -4,18 +4,20 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faUser, faCalendarAlt, faUsers, faHeart, faThumbsDown } from '@fortawesome/free-solid-svg-icons'
 
 
-class CardDisplaySearchResult extends React.Component {
+class CardResult extends React.Component {
+    state = { hovering: false }
+    mouseOver = () => this.setState({ hovering: true })
+    mouseOut = () => this.setState({ hovering: false })
+
     render() {
-        const { title, description, channelTitle, publishedAt } = this.props.youtubeItem.snippet;
-        const views = 19999;
-        const likes = 1200;
-        const disLikes = 200;
-        const urlImg = this.props.youtubeItem.snippet.thumbnails.medium.url;
+        const { title, description, channelTitle, publishedAt, urlImg, urlVideo, views, likes, disLikes } = this.props;
+        const hoverClassName = this.state.hovering ? 'card_hover mb-3 mt-3' : 'mb-3 mt-3';
 
         return (
-            <Card className="mb-3 mt-3">
+            <Card className={hoverClassName} onMouseOver={this.mouseOver} onMouseOut={this.mouseOut}>
```

Comment:
Why not use plain `:hover` selector?

Reply:
It's just for practice use events in a component. That's why.